packages/synapse-react-client/jest.config.cjs

```diff
@@ -3,6 +3,7 @@ const esModules = [
   '@react-hook',
   'lodash-es',
   'nanoid',
+  'mui-one-time-password-input'
```

Comment:
Upgraded `mui-one-time-password-input` to v2, which only emits ESM. Need to account for that in the Jest config to allow our tests to run.